packages/ui-extensions/src/surfaces/customer-account/components/Avatar/Avatar.doc.ts

```diff
@@ -10,7 +10,7 @@ const data: ReferenceEntityTemplateSchema = {
   type: '',
   definitions: [
     {
-      title: 'AvatarProps',
+      title: 'Properties',
```

Comment:
Both Checkout ([example](https://shopify.dev/docs/api/checkout-ui-extensions/2025-10-rc/polaris-web-components/overlay/modal#properties)) and Admin ([example](https://shopify.dev/docs/api/admin-extensions/2025-10-rc/polaris-web-components/feedback/banner#properties)) call this section "Properties". We should be aligned